alphamountain/package.yaml

```diff
@@ -81,7 +81,7 @@ pipelines:
       | where @name == "alphamountain.threat"
       | legacy "context update alphamountain-threats --key=url --create-timeout {{ inputs.threat-expiry }}"
 
-snippets:
+examples:
```

Comment:
Note that this needs to be merged only after the next release of the Tenzir Platform.